### PR TITLE
Adding randomwall and ollama

### PR DESCRIPTION
**Pull request by Soliprem**
- Changes to the switchwall script so that it can also take a positional $1 argument in the form of a wallpaper path
- Added randomwall script
- Added Ollama (with default model llama3:instruct. This could be changed in favour of a smaller model. Requires ollama to be installed and running on the system)
- Tested on my system with no issues

### CODE REVIEW

**Comment by Soliprem** [comment]
Not sure why the first commit reports a change to the hyprlock config file. Must have accidentally opened it and the linter might have added a space. No functional change, though

File: .config/ags/assets/icons/ollama.png
Comment (end-4) [inline .config/ags/assets/icons/ollama.png:1]: use svg
force push so the png doesn't take up space

File: .config/ags/scripts/color_generation/randomwall.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+XDG_CONFIG_HOME="${XDG_CONFIG_HOME:-$HOME/.config}"
+CONFIG_DIR="$XDG_CONFIG_HOME/ags"
+$CONFIG_DIR/scripts/color_generation/switchwall.sh $(fd . ~/Images/wallpapers/ -e .png -e .jpg -e .svg | xargs shuf -n1 -e)
```
Comment (end-4) [inline .config/ags/scripts/color_generation/randomwall.sh]: ~/Pictures not ~/Images

File: .config/ags/scripts/color_generation/switchwall.sh [hunk 2]
```diff
@@ -3,31 +3,35 @@
...
-        exit 0
-    fi
+	if [ "$imgpath" == '' ]; then
+		echo 'Aborted'
+		exit 0
+	fi
 
+	# ags run-js "wallpaper.set('')"
+	# sleep 0.1 && ags run-js "wallpaper.set('${imgpath}')" &
+	swww img "$imgpath" --transition-step 1 --transition-fps 120 \
```
Comment (end-4) [inline .config/ags/scripts/color_generation/switchwall.sh]: --transition-step 100 pls

**Review by end-4 — CHANGES_REQUESTED** [review]

File: .config/ags/scripts/color_generation/randomwall.sh (hunk 1 above)
Comment (end-4) [reply .config/ags/scripts/color_generation/randomwall.sh]: fixed

File: .config/ags/scripts/color_generation/switchwall.sh (hunk 2 above)
Comment (end-4) [reply .config/ags/scripts/color_generation/switchwall.sh]: fixed

File: .config/ags/assets/icons/ollama.png
Comment (end-4) [reply .config/ags/assets/icons/ollama.png:1]: done

Comment (end-4) [reply .config/ags/assets/icons/ollama.png:1]: force push removed
added svg in bf62babb29a027ba33ca9f15cb48e86ef17085f2

**Comment by Soliprem** [comment]
Thanks for the fixes! Those are preferences from my personal config and forgot I changed them

**Comment by end-4** [comment]
somehow this pr doubled the commit count lol
guess i shouldn't use git-filter-repo on a pr next time

**Pull request merged**